### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JukkaAtWork**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by Kozoft** [comment]
Review conversation attempt #3

**Review by igu007 — CHANGES_REQUESTED** [review]
codeowners doesn't work

File: README.md [hunk 1]
```diff
@@ -48,3 +48,5 @@ It is useful but not required to have a basic knowledge of the following topics:
 The provisioning of this environment for proctors can be found in the [DevOps Openhack Proctor](https://github.com/Azure-Samples/openhack-devops-proctor) Github repository.
 
 Testing that branch policies are enabled
+
+Testing the PR review experience and CODEOWNERS
```
Comment (TheDukeDK) [inline README.md:52]: Start a conversation...

File: README.md [hunk 2]
```diff
@@ -48,3 +48,5 @@ It is useful but not required to have a basic knowledge of the following topics:
 The provisioning of this environment for proctors can be found in the [DevOps Openhack Proctor](https://github.com/Azure-Samples/openhack-devops-proctor) Github repository.
```
Comment (Kozoft) [inline README.md:48]: Conversation

**Review by Kozoft — APPROVED** [review]

**Review by igu007 — APPROVED** [review]
Now it is complete

**Pull request merged**